src/extension.ts

```diff
@@ -30,7 +32,8 @@ export async function activate(context: ExtensionContext): Promise<AzureAccount>
 	context.subscriptions.push(commands.registerCommand('azure-account.openCloudConsoleWindows', () => cloudConsole(azureLoginHelper.api, 'Windows')));
 	context.subscriptions.push(commands.registerCommand('azure-account.uploadFileCloudConsole', uri => uploadFile(azureLoginHelper.api, uri)));
 	survey(context, reporter);
-	return Promise.resolve(azureLoginHelper.api); // Return promise to work around weird error in WinJS.
```

Comment:
This workaround was added 4 years ago in 6112d6dd75e6de51e568de69cc2ff12e0c8184bc so I feel comfortable removing it